apps/admin/integration-testing/cypress/support/auth.ts

```diff
@@ -1,14 +1,28 @@
 import { methodUrl } from '@votingworks/grout';
 
+// Importing all of @votingworks/auth causes Cypress tests to fail since Cypress doesn't seem to
+// interact well with PCSC Lite card reader code
+// eslint-disable-next-line vx/no-import-workspace-subfolders
+import {
+  mockCard,
+  MockFileContents,
+} from '@votingworks/auth/src/mock_file_card';
+
 const PIN = '000000';
 
+function mockCardCypress(mockFileContents: MockFileContents): void {
+  mockCard(mockFileContents, cy.writeFile);
+}
```

Comment:
I was running into this error when trying to use the base `mockCard` here: https://stackoverflow.com/questions/63797683/how-to-remove-fs-readfilesync-is-not-a-function-error-from-cypress-test

Using `cy.writeFile` instead of `fs.writeFileSync` did the trick

Reply:
Ah, I was wondering why you had a parameter for the `writeFileFn`, that explains it!

Reply:
Ah yeah, I should've left a note on the earlier commit! But glad it's all clear now